tests/plugins/mockserver/test_unix_mockserver.py

```diff
@@ -0,0 +1,92 @@
+import aiohttp
+import pytest
+
+from testsuite._internal import fixture_types
+from testsuite.daemons import service_client
+from testsuite.mockserver import classes
+from testsuite.mockserver import server
+
+
+@pytest.fixture
+def unix_mockserver(
+    _unix_mockserver: server.Server,
+    _mockserver_trace_id: str,
+):
+    with _unix_mockserver.new_session(_mockserver_trace_id) as session:
+        yield server.MockserverFixture(_unix_mockserver, session)
+
+
+@pytest.fixture(scope='session')
+async def _unix_mockserver(
+    testsuite_logger, _mockserver_reporter, pytestconfig
+):
+    async with server.create_unix_server(
+        '/tmp/mockserver.socket',
+        loop=None,
+        testsuite_logger=testsuite_logger,
+        mockserver_reporter=_mockserver_reporter,
+        pytestconfig=pytestconfig,
+    ) as result:
+        yield result
+
+
+@pytest.fixture(scope='session')
+def unix_mockserver_info(
+    _unix_mockserver: server.Server,
+) -> classes.MockserverUnixInfo:
+    return _unix_mockserver.server_info
+
+
+@pytest.fixture
+async def unix_mockserver_client(
+    unix_mockserver: fixture_types.MockserverFixture,
+    unix_mockserver_info: classes.MockserverUnixInfo,
+    pytestconfig,
+) -> service_client.Client:
+    conn = aiohttp.UnixConnector(path=unix_mockserver_info.socket_path)
+    session = aiohttp.ClientSession(connector=conn)
+
+    service_client_options = {
```

Comment:
use `service_client_options` fixture instead

Reply:
I made separate dict with manually set parameters in order to create `ClientSession` with custom `UnixConnector`.
`service_client_options` creates usual tcp session. I can write `unix_service_client_options` but after that I will need to create `unix_service_client_session` and `unix_service_client_session_factory` as well.